src/components/album/AlbumChatBox.js

```diff
@@ -78,6 +77,21 @@ const AlbumChatBox = ({ spotifyUrl }) => {
     };
   }, [albumId, socket, loggedInUserId]);
 
+  useEffect(() => {
+    const fetchRecentMessages = async () => {
+      if (albumId && !socket) {
+        const { data } = await axiosInstance.get(`/chat/${albumId}`);
+        console.log('Data from server:', data);
+        const messagesWithKeys = data.messages.map((message) => ({
+          ...message,
+          user: { name: message.user.name, id: message.user._id },
```

Comment:
I think now that the messages are coming in this shape already:

```json
{
    "_id": "6550fd4943fb620d4428ad30",
    "albumId": "6r1lh7fHMB499vGKtIyJLy",
    "message": "good and you?",
    "user": {
        "_id": "64f7d0f615a69acfaa074420",
        "name": "Akos999",
        "id": "64f7d0f615a69acfaa074420"
    },
    "timestamp": "2023-11-12T16:28:57.335Z",
    "__v": 0
}
```

I don't think we need line 86-87 where you're manually setting those values? 🤔  I think we can just pass in messages as-is

Reply:
> I think now that the messages are coming in this shape already:
> 
> ```json
> {
>     "_id": "6550fd4943fb620d4428ad30",
>     "albumId": "6r1lh7fHMB499vGKtIyJLy",
>     "message": "good and you?",
>     "user": {
>         "_id": "64f7d0f615a69acfaa074420",
>         "name": "Akos999",
>         "id": "64f7d0f615a69acfaa074420"
>     },
>     "timestamp": "2023-11-12T16:28:57.335Z",
>     "__v": 0
> }
> ```
> 
> I don't think we need line 86-87 where you're manually setting those values? 🤔 I think we can just pass in messages as-is

<img width="436" alt="1" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/95896159/4f8b0ed4-943b-443a-ba8d-d19702bd22ce">
removed this code